psl/psl-core/src/validate/validation_pipeline/validations/relation_fields.rs

```diff
@@ -151,8 +152,21 @@ pub(super) fn referential_actions(field: RelationFieldWalker<'_>, ctx: &mut Cont
     let msg_foreign_keys = |action: ReferentialAction| {
         let allowed_actions = connector.referential_actions();
 
+        let additional_info = match (action, connector.provider_name()) {
+            (ReferentialAction::SetDefault, "mysql") => Some(formatdoc! {
+                r#"
+                    . `{set_default}` is invalid for {provider} when using `relationMode = \"foreignKeys\"`, as {provider} does not support the `SET DEFAULT` referential action.
+                    Learn more at https://github.com/prisma/prisma/issues/11498
```

Comment:
This should already be a pris.ly link that redirects to this URL, so we can fix this later without a release. Jsut create the shortlink by adding to the vercel config file there via a PR.

Reply:
I haven't ever done that, could you please post here a link to a pris.ly README or something? I'll address that on Monday.

Reply:
Here: https://github.com/prisma/pris.ly/blob/main/vercel.json